scipy/stats/qmc.py

```diff
@@ -0,0 +1,647 @@
+"""Quasi-Monte Carlo methods.
+
+Define function to generate sample of points in the unit hypercube.
+
+"""
+
+from __future__ import division
```

Comment:
Python 2.x shouldn't matter in master branch anymore & backporting a new feature would be unusual--so this line can go, right?